src/main/Employee.java

```diff
@@ -7,8 +7,16 @@ public Employee(String name, double salary) {
         super(name, salary);
     }
 
-    @Override
+
     public boolean isSatisfied() {
-        return getSalary() > 10_000;
+        return this.getSalary() > 10000.0;
```

Comment:
- **Plusior** za 10000.0 warto dodawać tą kropkę odrazu widać że liczba jest double'm nie trzeba nawet patrzeć na typ.
- Co mógłbyś jeszcze zrobić to wyekstrahować tą wartość do stałej (alt + ctrl + c) bo w tym momencie to takie trochę _magic number_ i nazwać ją np. SATISFYING_SALARY wtedy odrazu byłoby widać o co chodzi, co więcej gdybyśmy użyli tej wartości jeszcze gdzieś w kodzie potem wystarczyłoby ją zmienić w jednym miejscu.